examples/platform/efr32/Service.cpp

```diff
@@ -106,3 +106,32 @@ void PublishService()
     }
     ThreadStackMgrImpl().UnlockThreadStack();
 }
+
+void StartDefaultThreadNetwork(void)
+{
+    // Set default thread network Info and enable the etWORK
```

Comment:
What is "etWORK"?

Reply:
Must have meant network and I had a stroke ;) Fixed.